extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientRuntimeConfig.java

```diff
@@ -128,6 +133,50 @@ public class InfinispanClientRuntimeConfig {
     @ConfigItem
     Optional<String> trustStoreType;
 
+    /**
+     * Configures caches from the client with the provided configuration.
+     */
+    @ConfigItem
+    Map<String, RemoteCacheConfig> cache = new HashMap<>();
+
+    @ConfigGroup
+    public static class RemoteCacheConfig {
+
+        /**
+         * Provides the configuration in XML to crete the cache on first access
+         */
+        @ConfigItem
+        public Optional<String> configuration;
```

Comment:
If not defined, is protostream used as in the examples below? Because if not, maybe we should make it the default as I understand that's what you would prefer with Quarkus?

Reply:
Protostream is the default marshaller in the client.